attributes/authoritative.rb

```diff
@@ -31,3 +31,8 @@
 default['pdns']['authoritative']['config']['guardian'] = true
 default['pdns']['authoritative']['config']['default_ttl'] = '3600'
 
+# This attribute is only required in authoritative package installs
+case node['pdns']['build_method']
```

Comment:
I would change this to a simple if statement versus a case since there is only one branch in this code. Also, is there is alternate way to set this because node is a fully rendered node object and prevents wrapping.